src/ontology/edits/oeo-physical.omn

```diff
@@ -6138,20 +6176,27 @@ Class: OEO_00030002
 
 In real world, fossils are from biogenic origin some thousands or millions of years ago. However, this is irrelevant in the energy modelling domain.",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/108
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/130",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/130
+
+Update definition:
+https://github.com/OpenEnergyPlatform/ontology/issues/872
+https://github.com/OpenEnergyPlatform/ontology/pull/1048",
         rdfs:label "fossil"
     
     SubClassOf: 
         OEO_00030003
     
     DisjointWith: 
-        OEO_00030004
+        
+            Annotations: <http://purl.obolibrary.org/obo/IAO_0000233> "https://github.com/OpenEnergyPlatform/ontology/issues/872
```

Comment:
The annotations of the 'DisjointWith' are a doubling because you already added it in the 'term tracker item'.
My understanding is, that we do not need it for the 'DisjointWith'.

Reply:
If we improve the term tracker item.